### PR TITLE
Add logs to worker

### PR DESCRIPTION
**Pull request by ArtoLord**

### CODE REVIEW

File: lzy/portal/src/main/resources/log4j2.yaml [hunk 1]
```diff
@@ -6,15 +6,13 @@ Configuration:
...
       target: SYSTEM_OUT
       PatternLayout:
-        Pattern: "%d{yyyy-MM-dd HH:mm:ss.SSS}{UTC} [%t] %-5level %logger{36} %notEmpty{[rid=%X{rid}] }- %msg%n"
-    File:
-      name: LogFile
-      fileName: ${sys:custom.log.file}
-      PatternLayout:
-        Pattern: "%d{yyyy-MM-dd HH:mm:ss.SSS}{UTC} [%t] %-5level %logger{36} %notEmpty{[rid=%X{rid}] }- %msg%n"
-
+        Pattern: "%d{yyyy-MM-dd HH:mm:ss.SSS}{UTC} [%t] %-5level %logger{36} - %msg%n"
```
Comment (imakunin) [inline lzy/portal/src/main/resources/log4j2.yaml:9]: plz, don't remove the `rid`

File: lzy/worker/SimpleWorker.Dockerfile [hunk 2]
```diff
@@ -39,31 +28,45 @@ RUN eval "$(conda shell.bash hook)" &&  \
...
 ENV JAVA_HOME=/opt/jre-minimal
 ENV PATH="$PATH:$JAVA_HOME/bin"
 
 COPY --from=conda-build /opt/conda/envs /opt/conda/envs
+COPY --from=conda-build /opt/conda/conda-meta /opt/conda/conda-meta
+
+# Running conda install to update metadata
+RUN conda install -n py39 stdlib-list=0.8.0=pyhd8ed1ab_0  \
+    && conda install -n py38 stdlib-list=0.8.0=pyhd8ed1ab_0  \
+    && conda install -n py37 stdlib-list=0.8.0=pyhd8ed1ab_0
```
Comment (imakunin) [inline lzy/worker/SimpleWorker.Dockerfile:51]: do we need python 10?

File: lzy/worker/SimpleWorker.Dockerfile [hunk 3]
```diff
@@ -39,31 +28,45 @@ RUN eval "$(conda shell.bash hook)" &&  \
...
+COPY docker/ua_config.yml /logging.yml
+
+RUN chmod +x /log_processor.py
+
+ENV LOGFILE="/tmp/worker.log"
+
+RUN echo '#!/bin/bash\n\
+set -m\n\
+java -Xmx4G -Dsun.jnu.encoding=UTF-8 -Dfile.encoding=UTF-8 -XX:+HeapDumpOnOutOfMemoryError -Djava.util.concurrent.ForkJoinPool.common.parallelism=32 \
+-Dmicronaut.env.deduction=true -Dio.grpc.netty.shaded.io.netty.eventLoopThreads=10 -Dio.grpc.netty.level=DEBUG -Dsun.net.level=DEBUG \
```
Comment (imakunin) [inline lzy/worker/SimpleWorker.Dockerfile:65]: why DEBUG?

File: lzy/worker/SimpleWorker.Dockerfile (hunk 2 above)
Comment (ArtoLord) [reply lzy/worker/SimpleWorker.Dockerfile:51]: We do not have py310 in this image now

File: lzy/worker/SimpleWorker.Dockerfile (hunk 3 above)
Comment (ArtoLord) [reply lzy/worker/SimpleWorker.Dockerfile:65]: It was here before)

Comment (ArtoLord) [reply lzy/worker/SimpleWorker.Dockerfile:65]: We are filtering this logs on log4j level

File: lzy/worker/docker/log_processor.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        message = json.loads(line)
+        payload = message["payload"]
+
+        parse = prog.match(payload)
+
+        meta = {
+            "timestamp": parse.group(1),
+            "thread": parse.group(2),
+            "level": parse.group(3),
+            "logger": parse.group(4)
```
Comment (imakunin) [inline lzy/worker/docker/log_processor.py:24]: where is `reqid`? :)

**Pull request merged**